### DISCUSSION TITLE
Inconsistent description text in (bitbucket) onboarding PR when using preset `:disableDependencyDashboard`

### DISCUSSION THREAD

**Discussion by jmartens** [renovatebot/renovate#32433] [Suggest an Idea]
### Tell us more.

When I onboard the repo with the following config:
```
  "extends": [
    "config:best-practices",
    ":disableDependencyDashboard",
    ":enablePreCommit",
    ":semanticCommits"
  ],
```

The description lists it will enable the dependency dashboard as well as disable it, see below for an example:

![image](https://github.com/user-attachments/assets/53147565-28ec-4507-896c-397776fa640b)

IMHO the first should not be there.

Not sure if this is for bitbucker onboarding PRs only, or that is (more) general.

**Comment by rarkins** [comment]
This would be a general problem and not Bitbucket specific. This text is autogenerated based in included presets and you have included one preset which enables it and another which disables it, so it's expected that both would be present.

If you feel strongly about this, you're welcome to propose a PR with a solution but otherwise this is not something worth our core team time to address
